diff --git a/node_modules/@revolist/vue-datagrid/src/vue-template.tsx b/node_modules/@revolist/vue-datagrid/src/vue-template.tsx
index a15053c..45b8a53 100644
--- a/node_modules/@revolist/vue-datagrid/src/vue-template.tsx
+++ b/node_modules/@revolist/vue-datagrid/src/vue-template.tsx
@@ -12,10 +12,11 @@ export const vueTemplateConstructor = (vueConstructor: VueConstructor, e: HTMLEl
 	if (e?.childNodes.length) {
 			el = e.childNodes[0] as VueElement;
 	}
-	
+	console.log(el);
 	if (!el) {
 		// create dom element wrapper for vue instance
-		el = document.createElement('span');
+
+		el = document.createElement('td');
 		e.appendChild(el);
 
 		// if passed as simple structure convert it to vue object
